code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -455,10 +458,12 @@
 		return 0
 
 	user.visible_message("<span class='warning'>[user] begins to dislocate [src]'s [organ.joint]!</span>")
+	user.limb_breaking = TRUE
 	if(do_after(user, 100))
 		organ.dislocate(1)
 		admin_attack_log(user, src, "dislocated [organ.joint].", "had his [organ.joint] dislocated.", "dislocated [organ.joint] of")
 		src.visible_message("<span class='danger'>[src]'s [organ.joint] [pick("gives way","caves in","crumbles","collapses")]!</span>")
+		user.limb_breaking = FALSE
 		return 1
 	user.visible_message("<span class='warning'>[user] fails to dislocate [src]'s [organ.joint]!</span>")

```

Comment:
I think you are missing a `user.limb_breaking = FALSE` here aswell.
Otherwise it's impossible to try and break someone's limbs again if you abort a limb breaking action as limb_breaking is still set to true

Reply:
Yep, you're right.